src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -157,6 +157,9 @@ angular
  * @param {string=} ng-pattern Adds the pattern validator to the ngModel of the search text.
  *     See the [ngPattern Directive](https://docs.angularjs.org/api/ng/directive/ngPattern)
  *     for more details.
+ * @param {string=} md-mode Specify the repeat-mode for suggestion lists. Acceptable values include
```

Comment:
Let's change "repeat-mode" to "repeat mode"

Reply:
Done :+1: